api/src/controllers/forms.js

```diff
@@ -95,4 +96,9 @@ module.exports = {
       })
       .catch(err => serverUtils.error(err, req, res));
   },
+  validate: (req, res) => {
+    return _generate(req.body)
```

Comment:
We use `_` to indicate a private function that shouldn't be used outside of the service. This was probably exported for testing purposes only. As this is a valid use of the function, you should rename it to `generate` and make sure it's documented and suitable for usage outside of the service.